src/pages/3rd-party/index.hbs

```diff
@@ -97,6 +97,11 @@ layout: main.hbs
             <td>A GWT wrapper for OpenLayers 3+ written in Java</td>
             <td><a href="https://github.com/TDesjardins">Tino Desjardins</a></td>
           </tr>
+          <tr>
+            <td><a href="https://github.com/terrestris/react-geo">react-geo</a></td>
+            <td>A set of geo related modules to use in combination with react, antd and OpenLayers</td>
```

Comment:
extra `antd`

Reply:
The antd word is actually a library…: https://github.com/ant-design/ant-design